portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase

```diff
@@ -127,8 +127,13 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8156 | Test Summary: Current AC plan level is shown in a card"
+	@ignore = "true"
 	@priority = 5
 	test CurrentACPlanLevelShownInCard {
+
+		// AC Bug: Current plans card is missing
+		// AC Bug ticket: LRAC-14001
+
```

Comment:
Hey @joshitagaki, shouldn't tests related to LRAC-14001 be re-enabled since the bug has been fixed?

Reply:
This wasn't merged into upstream yet. Looks like it was just merged in last night so we can probably remove this from being ignored now